helpdesk/settings.py

```diff
@@ -231,12 +228,24 @@
 HELPDESK_USE_HTTPS_IN_EMAIL_LINK = getattr(
     settings, 'HELPDESK_USE_HTTPS_IN_EMAIL_LINK', settings.SECURE_SSL_REDIRECT)
 
-HELPDESK_TEAMS_MODEL = getattr(
-    settings, 'HELPDESK_TEAMS_MODEL', 'pinax_teams.Team')
-HELPDESK_TEAMS_MIGRATION_DEPENDENCIES = getattr(settings, 'HELPDESK_TEAMS_MIGRATION_DEPENDENCIES', [
+# Default to True for backwards compatibility
+HELPDESK_TEAMS_MODE_ENABLED = False if hasattr(settings, 'HELPDESK_TEAMS_MODE_ENABLED') and getattr(
+    settings, 'HELPDESK_TEAMS_MODE_ENABLED', True) is False else True
```

Comment:
Same, we can make it more readable:

```suggestion
HELPDESK_TEAMS_MODE_ENABLED = getattr(settings, 'HELPDESK_TEAMS_MODE_ENABLED', True)
```